Dockerfile

```
@@ -0,0 +1,7 @@
+FROM python:3
+ENV PYTHONUNBUFFERED 1
+RUN apt-get update && apt-get install -y python-dev libldap2-dev libsasl2-dev libssl-dev
```

`gettext` is needed to be able to generate translations.

```suggestion
RUN apt-get update && apt-get install -y python-dev libldap2-dev libsasl2-dev libssl-dev gettext libgettextpo-dev
```